packs/data/kingmaker-bestiary.db/foras.json

```diff
@@ -6025,29 +6025,7 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [
```

Comment:
Add this back. There is value in drawing the circle as a GM hint even if the granted item doesn't apply.